extensions/standard-processors/tests/unit/ProcessorTests.cpp

```diff
@@ -180,19 +184,37 @@ TEST_CASE("Test GetFile Ignore", "[getfileCreate3]") {
   auto records = reporter->getEvents();
   record = session->get();
   REQUIRE(record == nullptr);
-  REQUIRE(records.size() == 0);
+  REQUIRE(records.empty());
 
-  std::fstream file;
-  std::stringstream ss;
-  ss << dir << utils::file::FileUtils::get_separator() << ".filewithoutanext";
-  file.open(ss.str(), std::ios::out);
-  file << "tempFile";
-  file.close();
+  const std::string hidden_file_name = [&] {
+    std::stringstream ss;
+    ss << dir << utils::file::FileUtils::get_separator() << ".filewithoutanext";
+    return ss.str();
+  }();
+  {
+    std::ofstream file{ hidden_file_name };
+    file << "tempFile";
+  }
+
+#ifdef WIN32
+  {
```

Comment:
This is inherently specific to Windows, since on windows "hidden" is a file system attribute while on unix it's a filename prefix. Because of the wildly different semantics, I would only add it inside an `#ifdef WIN32`/`#endif /* WIN32 */` there as well.

Would that be good in your opinion?

Reply:
Yeah, that's fine.

Reply:
I also used this Win API call in GetFileTests.cpp. In case you create an util func, please make that depend on that, too!